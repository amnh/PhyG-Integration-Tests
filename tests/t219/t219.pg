--fuse(once), inputting fastc files along with tcm
set(seed:1634561640)
read(rename: "woman_rename.txt")
--read(fastc:"woman.fastc", tcm: "woman-11.tcm")
read(fastc:"woman.fastc")
read(fastc:"woman_cox1_spaced.fastc")
read(include:"Include_woman16.txt")
read(newick:"../t80/t80.tre")
select(best:2)
fuse(once, pairs:1)
report("t219.csv", data, overwrite)
report("t219.tre", newick, graphs, overwrite)