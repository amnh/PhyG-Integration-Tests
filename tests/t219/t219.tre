(AlbanianTosk:72.5,((ArmenianWestern:33.0,(Scots:31.0,(OldPrussian:31.5,(Faroese:25.5,Icelandic:58.5)HTU17:18.5)HTU18:18.0)HTU19:15.0)HTU20:18.0,(Danish:36.5,(EasternFrisianSaterland:26.0,((KolschColognian:42.0,Yiddish:30.0)HTU28:22.0,((Irish:24.5,(AncientIonicGreek:31.5,(Assamese:29.0,SanskritVedic:17.5)HTU24:41.5)HTU25:22.0)HTU26:18.5,(Cornish:28.5,Manx:21.5)HTU27:23.0)HTU30:15.0)HTU29:12.5)HTU22:18.5)HTU23:26.5)HTU21:60.0)HTU16[729.0];
(AlbanianTosk:64.5,(ArmenianWestern:33.0,(Danish:34.0,(((Manx:16.5,(Cornish:25.5,(Irish:34.5,(AncientIonicGreek:35.5,(Assamese:24.0,SanskritVedic:21.0)HTU22:29.5)HTU23:25.5)HTU24:20.5)HTU25:14.0)HTU26:29.0,(Scots:32.0,(EasternFrisianSaterland:25.5,(KolschColognian:39.5,Yiddish:32.5)HTU20:25.5)HTU21:15.0)HTU27:13.5)HTU28:11.0,(OldPrussian:34.5,(Faroese:27.0,Icelandic:57.0)HTU30:18.0)HTU29:10.5)HTU18:8.5)HTU19:20.5)HTU17:38.5)HTU16[729.0];
(AlbanianTosk:29.5,(Danish:35.5,((OldPrussian:32.0,(Faroese:26.5,Icelandic:57.5)HTU17:21.5)HTU18:10.5,(ArmenianWestern:34.0,(Scots:34.0,((EasternFrisianSaterland:28.5,((KolschColognian:43.0,Yiddish:29.0)HTU25:22.5,(Irish:26.5,(AncientIonicGreek:33.5,(Assamese:28.0,SanskritVedic:18.5)HTU23:31.0)HTU24:26.0)HTU26:19.5)HTU27:11.5)HTU28:13.0,(Cornish:28.0,Manx:22.0)HTU29:23.5)HTU30:16.0)HTU21:13.0)HTU22:12.0)HTU19:9.5)HTU20:30.0)HTU16[729.0];
(AlbanianTosk:73.5,(ArmenianWestern:32.0,(Danish:30.0,((Cornish:31.5,Manx:18.5)HTU26:19.0,((Scots:30.5,(EasternFrisianSaterland:25.5,((KolschColognian:41.5,Yiddish:30.5)HTU22:23.0,(Irish:24.0,(AncientIonicGreek:31.0,(Assamese:29.0,SanskritVedic:18.0)HTU20:43.5)HTU21:24.0)HTU23:21.0)HTU24:14.5)HTU25:18.5)HTU27:23.5,(OldPrussian:29.0,(Faroese:26.0,Icelandic:58.0)HTU28:21.0)HTU29:14.5)HTU30:12.5)HTU18:11.0)HTU19:24.0)HTU17:57.5)HTU16[729.0];
(AlbanianTosk:73.5,(ArmenianWestern:33.0,(Danish:33.5,((Scots:30.0,(EasternFrisianSaterland:25.5,((KolschColognian:41.5,Yiddish:30.5)HTU22:23.5,(Irish:24.0,(AncientIonicGreek:31.0,(Assamese:29.0,SanskritVedic:18.0)HTU20:43.5)HTU21:24.0)HTU23:21.0)HTU24:15.5)HTU25:18.0)HTU27:21.0,((Cornish:31.0,Manx:19.0)HTU26:19.0,(OldPrussian:28.5,(Faroese:27.5,Icelandic:56.5)HTU28:23.5)HTU29:10.5)HTU30:13.5)HTU18:12.0)HTU19:19.5)HTU17:58.0)HTU16[729.0];
(AlbanianTosk:29.5,(Danish:35.5,((OldPrussian:32.5,(Faroese:26.0,Icelandic:58.0)HTU17:21.5)HTU18:10.5,(ArmenianWestern:31.5,(Scots:33.0,((EasternFrisianSaterland:27.5,(KolschColognian:42.0,Yiddish:30.0)HTU27:23.5)HTU28:15.5,((Cornish:28.0,Manx:22.0)HTU25:19.5,(Irish:28.5,(AncientIonicGreek:34.0,(Assamese:28.5,SanskritVedic:18.5)HTU23:30.0)HTU24:22.0)HTU26:16.5)HTU30:16.0)HTU29:17.0)HTU21:16.5)HTU22:13.0)HTU19:10.5)HTU20:30.0)HTU16[728.0];
(AlbanianTosk:29.5,(Danish:35.5,((OldPrussian:32.5,(Faroese:26.0,Icelandic:58.0)HTU20:21.5)HTU21:10.5,(ArmenianWestern:31.5,(Scots:33.0,((EasternFrisianSaterland:27.5,(KolschColognian:42.0,Yiddish:30.0)HTU17:23.5)HTU18:15.5,((Irish:28.5,(AncientIonicGreek:34.0,(Assamese:28.5,SanskritVedic:18.5)HTU27:30.0)HTU28:22.0)HTU29:16.5,(Cornish:28.0,Manx:22.0)HTU30:19.5)HTU25:16.0)HTU26:17.0)HTU19:16.5)HTU22:13.0)HTU23:10.5)HTU24:30.0)HTU16[728.0];
(AlbanianTosk:73.5,(ArmenianWestern:36.5,((Danish:30.5,(Icelandic:55.5,(Faroese:30.5,OldPrussian:35.0)HTU17:20.0)HTU18:16.5)HTU19:20.5,(Scots:32.0,((EasternFrisianSaterland:25.0,((KolschColognian:43.0,Yiddish:29.0)HTU25:23.5,(Irish:24.0,(AncientIonicGreek:29.0,(Assamese:29.5,SanskritVedic:17.0)HTU23:45.0)HTU24:24.5)HTU26:21.0)HTU27:12.0)HTU28:15.5,(Cornish:28.0,Manx:22.0)HTU29:23.0)HTU30:16.0)HTU21:14.5)HTU22:18.5)HTU20:59.5)HTU16[727.0];
(AlbanianTosk:67.0,(ArmenianWestern:32.5,(Danish:31.0,((Manx:16.0,(Cornish:24.0,(Irish:35.5,(AncientIonicGreek:35.0,(Assamese:24.0,SanskritVedic:21.0)HTU18:30.0)HTU19:25.5)HTU20:20.5)HTU21:17.0)HTU22:29.5,(OldPrussian:29.5,(Icelandic:58.0,(Faroese:18.5,((KolschColognian:39.5,Yiddish:32.5)HTU29:26.0,(EasternFrisianSaterland:30.5,Scots:32.5)HTU30:13.5)HTU27:20.0)HTU28:19.5)HTU17:20.5)HTU23:9.0)HTU24:8.0)HTU25:24.5)HTU26:37.5)HTU16[725.0];
(AlbanianTosk:71.0,(ArmenianWestern:37.0,(Scots:26.5,((Icelandic:59.0,(Faroese:29.5,OldPrussian:35.5)HTU17:13.0)HTU18:15.0,(Danish:29.5,((Cornish:29.0,Manx:21.0)HTU26:21.5,((Irish:29.5,(AncientIonicGreek:32.0,(Assamese:24.0,SanskritVedic:21.0)HTU24:43.0)HTU25:25.0)HTU27:21.0,(EasternFrisianSaterland:26.5,(KolschColognian:41.5,Yiddish:30.5)HTU28:23.0)HTU29:16.5)HTU30:19.5)HTU22:18.0)HTU23:22.5)HTU19:19.0)HTU20:13.5)HTU21:58.5)HTU16[725.0];
