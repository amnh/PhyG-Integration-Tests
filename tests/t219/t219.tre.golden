(AlbanianTosk:45.0,(Danish:32.0,(((OldPrussian:26.0,(Faroese:18.0,Icelandic:50.0)HTU21:9.0)HTU20:1.0,(((Cornish:23.0,Manx:14.0)HTU24:14.0,(EasternFrisianSaterland:20.0,Yiddish:36.0)HTU25:6.0)HTU23:1.0,(Assamese:50.0,Scots:24.0)HTU26:4.0)HTU22:4.0)HTU19:0.0,(ArmenianWestern:18.0,(KolschColognian:36.0,(SanskritVedic:21.0,(AncientIonicGreek:34.0,Irish:28.0)HTU30:9.0)HTU29:6.0)HTU28:6.0)HTU27:8.0)HTU18:7.0)HTU17:18.0)HTU16[773.0];
(AlbanianTosk:38.0,(Danish:36.0,(((OldPrussian:26.0,(Faroese:17.0,Icelandic:52.0)HTU21:9.0)HTU20:1.0,(((Irish:18.0,(Cornish:22.0,Manx:17.0)HTU25:14.0)HTU24:5.0,(Yiddish:21.0,(EasternFrisianSaterland:25.0,KolschColognian:40.0)HTU27:6.0)HTU26:3.0)HTU23:4.0,(Assamese:35.0,Scots:28.0)HTU28:6.0)HTU22:3.0)HTU19:13.0,(ArmenianWestern:21.0,(AncientIonicGreek:42.0,SanskritVedic:17.0)HTU30:15.0)HTU29:5.0)HTU18:8.0)HTU17:8.0)HTU16[769.0];
